Products/CMFPlone/interfaces/controlpanel.py

```diff
@@ -545,7 +545,7 @@ class ITinyMCELayoutSchema(Interface):
             u'Enter a JSON-formatted style format configuration. '
             u'A format is for example the style that get applied when '
             u'you press the bold button inside the editor. '
-            u'See http://www.tinymce.com/wiki.php/Configuration:formats'),
+            u'See https://www.tinymce.com/wiki.php/Configuration:formats'),
```

Comment:
this links redirects to `http://www.tinymce.com/docs/configure/content-formatting/#formats`